README.md

```diff
@@ -13,5 +13,7 @@ Example actors, capability providers, and other demonstrations
 | [subscriber](https://github.com/wasmcloud/examples/tree/main/subscriber) | Actor | A simple actor that logs every message it receives to `stdout` | wasmcloud.azurecr.io/subscriber:0.2.0 |
 | [inmemory-keyvalue](https://github.com/wasmcloud/examples/tree/main/inmemory-keyvalue) | Provider | A sample in-memory Key-Value Store capability provider, used by the tutorial for creating a new capability provider | wasmcloud.azurecr.io/inmemory-keyvalue:0.4.0 |
 | [inmemory-streams](https://github.com/wasmcloud/examples/tree/main/inmemory-streams) | Provider | An example illustrating an in-memory `wasmcloud:eventstreams` provider | Not published |
+| [actor-to-actor](https://github.com/wasmcloud/examples/tree/main/actor-to-actor) | Actors | An example illustrating shared actor interface and actor-to-actor communication | Not Published
```

Comment:
Nitpick, thoughts on moving this up to two spaces in the table so it can nestle below the "Actor" examples?

Reply:
done